requirements.txt

```
@@ -136,7 +136,7 @@ smartypants==2.0.1
     # via notifications-utils
 statsd==3.3.0
     # via notifications-utils
-urllib3==1.26.4
+urllib3==1.26.5
```

I’d suggest using the [same (newer) version as the API](https://github.com/alphagov/notifications-api/blob/318cc1284aefa6f8e30928c1a0f703d5e5cc3a59/requirements.txt#L249) because we know that is running fine in production:

```suggestion
urllib3==1.26.7
```

---

Good point, that's now updated 👍